EIPS/eip-663.md

```diff
@@ -1,8 +1,8 @@
 ---
 eip: 663
-title: Unlimited SWAP and DUP instructions
-description: Introduce SWAPN and DUPN which take an immediate value for the depth
-author: Alex Beregszaszi (@axic)
+title: Additional stack manipulation instructions
+description: Introduce additional stack manipulation instructions which take an immediate arguments for the depth
+author: Alex Beregszaszi (@axic), Charles Cooper (@charles-cooper), Danno Ferrin (@shemnon)
 discussions-to: https://ethereum-magicians.org/t/eip-663-unlimited-swap-and-dup-instructions/3346
 status: Review
```

Comment:
3540 got marked as stagnant (will be fixed soon). Can we in the meantime put this back to `Draft`? Otherwise we can't merge the PR :(